---
title: "Bionformatics Resources"
page-layout: full
---

## General Bioinformatics Resources

Depending on your project, you may find some of the following resources useful - please note that this is by no means an exhaustive list, just a starting point for your work - there are many more helpful resources out there. Feel free to discuss with Dr. Feeney if you have any questions.

-   [How to Access Data from Public Databases](https://sipbs-compbiol.github.io/BM432/notebooks/02-03-public_databases.html)

-   [Links to some key public databases](https://sipbs-compbiol.github.io/BM432/notebooks/database_list.html).

-   [Get Started with GitHub documentation](https://docs.github.com/en/get-started)

-   [Galaxy Video Lectures](https://www.youtube.com/c/galaxyproject)

## Useful tools/websites

-   [Interactive Tree of Life (iTOL)](https://itol.embl.de/): Useful for visualising phylogenetic trees, and editing their presentation

-   [ExPASy resource portal](https://www.expasy.org/): A suite of different bioinformatic resources

-   [JalView](https://www.jalview.org/): multiple sequence alignment editing, visualisation and analysis

-   [PyMol](https://pymol.org/2/): 3D molecular visualisation

## Streptomyces/actinomycetes resources

-   [StrepDB](StrepDB:%20http://strepdb.streptomyces.org.uk/cgi-bin/dc3.pl?accession=AL645882&start=4291472&end=4302043&iorm=map&width=900)

-   [Tables of Streptomyces coelicolor reciprocal BLASTP hits](https://streptomyces.org.uk/actinoblast/)

-   [antiSMASH](https://antismash.secondarymetabolites.org/#!/start)

-   [ActinoBase](https://actinobase.org/index.php/Main_Page)

## Phage resources

- [Actinobacteriophage Database](https://phagesdb.org/)

- [Phage tutorial 1](./assets/Galaxy_instructions_annotate_phage_genome.docx): In this first tutorial, we go through how to create and name a new history in Galaxy, identify and import phage genome sequences into Galaxy, and annotate and examine the imported phage genomes

- [Phage tutorial 2](./assets/Galaxy_instructions_phylogenetic_tree.docx): In this tutorial, we work through how to create a phylogenetic tree for a gene/protein of interest

- [Phage tutorial 3](./assets/Galaxy_instructions_downstream_analyses.docx): in this tutorial, we go through some of the possible downstream analyses that can be done once you have identified gene(s) or protein(s) of interest in your phage genome, including: BLAST analysis, structural prediction/analysis, analysis of physicochemical properties, and codon usage calculations

- [Phage tutorial 4](./assets/Galaxy_instructions_hostside.docx): This tutorial looks at different ways of analysing the host genomes, including BLAST analysis, searching for CRISPR-Cas systems and other antiphage defenses, and analysis of secondary (specialised) metabolite production

## actDES resources

-   Schniete, J. K., Selem-Mojica, N., Birke, A. S., Cruz-Morales, P., Hunter, I. S., Barona-Gomez, F., & Hoskisson, P. A. (2021). ActDES - a curated Actinobacterial Database for Evolutionary Studies. Microbial genomics, 7(1), mgen000498. https://doi.org/10.1099/mgen.0.000498

-   [e-seminar explaining actDES](https://youtu.be/iSlrPdI4ohU)

-   [actDES GitHub repository](https://github.com/nselem/ActDES)

-   [actDES FigShare page](https://microbiology.figshare.com/articles/dataset/ActDES-_a_Curated_Actinobacterial_Database_for_Evolutionary_Studies/13143407/1)

-   [actDES Python script](https://github.com/PaulHoskisson/StrepEvo/blob/main/ActDES_python) and [YouTube walkthrough](https://www.youtube.com/watch?v=OdeuV9Iqw6M)

## Additional selected tutorials

- links to Leighton's stuff here